properties/P000101.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: P000101
+slug: rc
+name: Retracts are closed
+aliases:
+  - RC
+refs:
+  - mo: 191016
+    name: '"All retracts are closed" as separation axiom'
+  - mo: 434451
+    name: '"All retracts are closed" and "all compacts are closed"'
+---
+
+Every retract subspace ($A\subseteq X$ with a continuous $f:X\to A$ constant on $A$) is closed.
```

Comment:
"constant on $A$" should probably be something like "fixing every element of $A$".